src/element.ts

```diff
@@ -393,14 +393,14 @@ export class Element {
    * 'bold 10pt Arial'.
    */
   getFont(): string {
-    return Font.toCSSString(this.textFont);
+    return Font.toCSSString(this.#textFont);
   }
 
   /** Return a copy of the current FontInfo object. */
   get fontInfo(): Required<FontInfo> {
     // We can cast to Required<FontInfo> here, because
-    // we just called resetFont() above to ensure this.textFont is set.
-    return this.textFont;
+    // we just called resetFont() above to ensure this.fontInfo is set.
+    return this.#textFont;
```

Comment:
If you're going to rename textFont in the subclasses... might as well do it for Element. It's a private var anyways.

`this.#textFont` => `this.#fontInfo`

Reply:
done